TricksterBots/Bots/FiveHundred/FiveHundredBot.cs

```diff
@@ -354,18 +359,23 @@ private Card TryBustNullo(PlayerBase player, IReadOnlyList<Card> trick, IReadOnl
 
                 foreach (var suit in SuitRank.stdSuits)
                 {
-                    //  avoid leading a suit any nullo player is void in
                     if (nulloPlayers.Any(p => players.TargetIsVoidInSuit(player, p, suit, cardsPlayed)))
                         avoidSuits.Add(suit);
                 }
 
-                var preferredLegalCards = legalCards.Where(c => !avoidSuits.Contains(EffectiveSuit(c))).ToList();
+                var knownCards = cardsPlayed.Concat(legalCards).ToList();
+                var preferredLegalCards = legalCards
+                    .Where(c => !avoidSuits.Contains(EffectiveSuit(c))) //  avoid leading a suit any nullo player is void in
+                    .Where(c => !IsCardHigh(c, knownCards)) //  also avoid leading a card that is known to be high
```

Comment:
When falling back, prefer trying again to pick a card that isn't high (but is in a previously avoided suit). This gives us the best chance to transfer the lead to our partner who may still be able to get under the nullo bidder.